### ISSUE TITLE
Scrolling in one window makes the other window scroll too (same file in vsplit)

### ISSUE THREAD

**Issue by davidbrochart** [jonathanslenders/pyvim#111]
From the pyvim page, this limitation still exists with prompt-toolkit 2.0:

"prompt_toolkit does not (or not yet) allow buffers to have an individual cursor when buffers are opened in several windows. Currently, this results in some unexpected behaviors, when a file is displayed in two windows at the same time. (The cursor could be displayed in the wrong window and other windows will sometimes scroll along when the cursor moves.) This has to be fixed in the future."

**Comment by jonathanslenders** [comment]
Indeed, support for multiple cursors is not yet there. This is not obvious to implement and has never been a priority.

However, when we have this, we can also have support for folding, key bindings that memorize cursor positions and much more.
